### ISSUE TITLE
PDF reader: Match on hyphenated text

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2978]
https://forums.zotero.org/discussion/102331/search-function-in-the-pdf-reader

Possible to do this? I suspect most PDF readers don't do this (e.g., Preview doesn't seem to), but since pdftotext does, there's a reasonable expectation that you'd be able to find the same text in the PDF.

**Comment by mrtcode** [comment]
That already works in the current PDF.js version. We'll update once Zotero 7 is out.

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/102550/end-of-line-hyphens-in-ocred-pdfs

Copying too, or is that separate?

**Comment by mrtcode** [comment]
Yeah, those are different things. And we already automatically remove hyphens. The reported issue with spaces, normally, doesn't happen, but I'll investigate. OCRed PDFs can have weird issues.
